src/reason_pprint_ast.ml

```diff
@@ -2218,10 +2218,9 @@ let formatAttachmentApplication finalWrapping (attachTo: (bool * layoutNode) opt
           | (hd::tl, Some (useSpace, toThis)) ->
             (* TODO: Can't attach location to this - maybe rewrite anyways *)
             let attachedArgs = makeAppList attachedList in
-            label
-              ~space:true
-              (label ~space:useSpace toThis attachedArgs)
-              wrappedListy
+              (label ~space:useSpace toThis (label
```

Comment:
I wonder if the real underlying cause of the comment formatting was due to a lack of `SourceMap` node that was needed.